src/nccl_ofi_rdma.c

```diff
@@ -5650,7 +5651,8 @@ static inline int init_bounce_buffers(nccl_net_ofi_rdma_ep_t *ep)
 	int ret = 0;
 
 	ep->bounce_buff_reqs_fl = NULL;
-	ep->bounce_buff_fl = NULL;
+	ep->bounce_buff_ctrl_fl = NULL;
```

Comment:
not really sure I understand all this.  Why are you not just creating the free lists?

Reply:
This is for the error case cleanup code below, so we know which freelists we have allocated (and need to be finalized). It's the same pattern we use in other places in the code to do error cleanup.

In this case, we don't really need to set these to NULL first since the endpoint is `calloc`'d (so they should already be NULL), but I think it's safer not to rely on that.

Reply:
If calloc's broken, we're so very broken that this won't matter.

Reply:
I refactored this cleanup code as part of moving freelists to rails.